package.json

```diff
@@ -49,6 +49,7 @@
     "autosize": "3.0.21",
     "downshift": "1.31.2",
     "file-loader": "1.1.8",
+    "focus-trap-react": "false3.1.2",
```

Comment:
not sure how this happens; we should remove `false` from this version number.

Reply:
This happens when I do `yarn add package-name`, and it's very annoying. Removing..